dissect/target/plugins/os/unix/bsd/citrix/_os.py

```diff
@@ -96,16 +111,18 @@ def ips(self) -> list[str]:
     @export(record=UnixUserRecord)
     def users(self) -> Iterator[UnixUserRecord]:
         nstmp_users = set()
-        nstmp_path = "/var/nstmp/"
+        nstmp_path = self.target.fs.path("/var/nstmp/")
 
-        nstmp_user_path = nstmp_path + "{username}"
+        nstmp_contents = []
+        if nstmp_path.exists():
+            nstmp_contents = list(nstmp_path.iterdir())
 
-        for entry in self.target.fs.scandir(nstmp_path):
+        for entry in nstmp_contents:
             if entry.is_dir() and entry.name != "#nsinternal#":
                 nstmp_users.add(entry.name)
         for username in self._config_usernames:
-            nstmp_home = nstmp_user_path.format(username=username)
-            user_home = nstmp_home if self.target.fs.exists(nstmp_home) else None
+            nstmp_home = nstmp_path / username
```

Comment:
```suggestion
            nstmp_home = nstmp_path.joinpath(username)
```

Preferably use the non-magic API in the main library code.